test/extension-tests/successful-build/test/debugger.test.ts

```diff
@@ -185,12 +185,17 @@ suite('Debug/Launch interface', () => {
         const exists = fs.existsSync(createdFileOnExecution);
         // Check that it is compiled as a new file
         expect(exists).to.be.true;
+
```

Comment:
This is a good idea when it comes to not duplicating code, however, it would possibly add test time to tests that don't need this fix, and I'd rather avoid that. Thanks!